usaspending_api/api_docs/markdown/data_dictionary.md

```diff
@@ -98,12 +98,14 @@ In this section you will find definitions for common terms used in the API. For
 | /api/v1/awards/:id | GET, POST | <a href="#award">Award</a> | Returns a single award records with all fields |
 | [/api/v1/awards/autocomplete/](/api/v1/awards/autocomplete/) | POST | Autocomplete (see [Using the API](/docs/using-the-api))| Supports autocomplete on award records |
 | [/api/v1/awards/total/](/api/v1/awards/total/) | POST |  Aggregate (see [Using the API](/docs/using-the-api)) | Supports aggregation on award records |
-| [/api/v1/accounts/balances/](/api/v1/accounts/balances/) | GET, POST | <a href="#accounts">Appropriation Account Balances</a> | Returns a list of appropriation account balances |
-| [/api/v1/accounts/balances/total/](/api/v1/accounts/balances/total/) | POST |  Aggregate (see [Using the API](/docs/using-the-api)) | Supports aggregation on appropriation account records |
-| [/api/v1/accounts/categories/](/api/v1/accounts/categories/) | GET, POST | <a href="#accounts-prg-obj">Appropriation Account Balances (by Category)</a> | Returns a list of appropriation account balances broken up by program activities and object class |
-| [/api/v1/accounts/categories/total/](/api/v1/accounts/categories/total/) | POST |  Aggregate (see [Using the API](/docs/using-the-api)) | Supports aggregation on appropriation account (by category) records |
-| [/api/v1/accounts/tas/](/api/v1/accounts/tas/) | GET, POST | <a href="#tas">Treasury Appropriation Account</a> | Returns a list of treasury appropriation accounts, by TAS |
-| [/api/v1/accounts/tas/autocomplete/](/api/v1/accounts/tas/autocomplete/) | POST | Autocomplete (see [Using the API](/docs/using-the-api))| Supports autocomplete on TAS records |
+| [/api/v1/federal_accounts/](/api/v1/federal_accounts/) | GET, POST | <a href="#federal-account">Federal Account</a> | Returns a list of federal accounts |
+| [/api/v1/federal_accounts/total/](/api/v1/federal_accounts/total/) | POST |  Aggregate (see [Using the API](/docs/using-the-api)) | Supports aggregation on federal accounts records |
```

Comment:
Ack, that was supposed to be /autocomplete, but I goofed a copy and paste. Thanks for pointing it out